From 81ea117e0396274808a77d40bf11af44ea047285 Mon Sep 17 00:00:00 2001
From: Rosen Penev <rosenp@gmail.com>
Date: Mon, 23 Sep 2024 12:01:40 +0200
Subject: [PATCH] pkgconfig: adjust for static libraries

OpenWrt builds only static libuuid for tools. Adjust the pkgconfig file
accordingly.

Addresses: https://github.com/util-linux/util-linux/issues/3210
Signed-off-by: Rosen Penev <rosenp@gmail.com>
---
 libuuid/uuid.pc.in | 2 +-
 1 file changed, 1 insertions(+), 2 deletions(-)

--- a/libuuid/uuid.pc.in
+++ b/libuuid/uuid.pc.in
@@ -6,6 +6,5 @@ includedir=@includedir@
 Name: uuid
 Description: Universally unique id library
 Version: @LIBUUID_VERSION@
-Requires:
 Cflags: -I${includedir}/uuid
-Libs: -L${libdir} -luuid
+Libs: -L${libdir} -luuid -lpthread
